about.html

```diff
@@ -0,0 +1,161 @@
+                                This year, find out what's on the horizon for streaming, esports, digital video creation, 
+                                investment, connected entertainment, and more.
+                            </p>
+                        </div>
+                        <div class="infos">
+                            <p class="location light-black text-center">
+                                <i>Please contact us if you need any questions about our streaming events!</i>
+                                <br />
+                                Email us at <b><a href="streaming2023@laxact.net">streaming2023@laxact.net</a></b>
+                                <br>or call us at <b>+246 598 4995</b>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </header>
+            <hr class="full-horizontal-line" />
+            <section class="about-us flex-column-center text-center" id="about-us">
+                <h2 class="about-us-head light-black heading-medium">About Us</h2>
+                <hr class="horizontal-line text-center" />
+                <img src="https://securityconference.org/media-library/images/resized/e/b/csm_20200214_welcomeRemarks___4281__1__eb24e7b010.jpg" alt="about us group-photo" class="group-photo" />
+                <div class="about-us-content">
+                    <p class="about-us-paragraph">
+                        laxACT, the next-generation streaming company, develops problem-solving strategies 
+                        and uses the latest IT trends to drive growth for businesses all over the world. 
+                        It has 15 different business units that offer IT solutions including CRM development, 
+                        Web Design, Application Development, Online Education, Research & Standardization Unit 
+                        and SAAS. Products of laxACT increase the efficiency of any organization by up to 300%.
+                    </p>
+                    <h4>
+                        <br />
+                        What do we promise to media providers and streaming enthusiasts?
+                    </h4>
+                    <p>
+                        Without question, analysts say, the flood of new streaming services will 
+                        cause more people to cancel traditional cable subscriptions. Cable television 
+                        is still the entertainment industry's cash cow, but millions of customers in the 
+                        United States have already cut the cord. The annual pace of subscriber decline hit 
+                        5.4 percent in the second quarter, a statistic Craig Moffett, a senior analyst at 
+                        MoffettNathanson, referred to in a recent report as “freaking ugly.”
+                    </p>
+                </div>
+            </section>
```

Comment:
I'd kindly suggest you follow the design for this section of your work, as there seems to be a big difference between your design and the one given. this section seems to be really length in your design and the text is too big, it is supposed to be just a logo and few write-up about the logo 👍 

Your design :

https://github.com/katarighe/microverse-capstone-project-1/assets/20537712/878e97e5-ca79-49f2-8d5b-4a58d7025a09

Given design :
![image](https://github.com/katarighe/microverse-capstone-project-1/assets/20537712/0e6a942b-1bec-4f2a-99bb-cabf8c0fb779)